packages/cubejs-questdb-driver/test/QuestQuery.test.ts

```diff
@@ -0,0 +1,172 @@
+import { QuestQuery } from '../src/QuestQuery';
+import { prepareCompiler as originalPrepareCompiler } from '@cubejs-backend/schema-compiler';
+
+const prepareCompiler = (content: string, options: any[]) => originalPrepareCompiler({
+  localPath: () => __dirname,
+  dataSchemaFiles: () => Promise.resolve([
+    { fileName: 'main.js', content }
+  ])
+}, { adapter: 'postgres', ...options });
+
+describe('QuestQuery', () => {
+  const { compiler, joinGraph, cubeEvaluator } = prepareCompiler(`
+    cube(\`visitors\`, {
+      sql: \`
+      select * from visitors
+      \`,
+
+      measures: {
+        count: {
+          type: 'count'
+        },
+
+        unboundedCount: {
+          type: 'count',
+          rollingWindow: {
+            trailing: 'unbounded'
+          }
+        }
+      },
+
+      dimensions: {
+        createdAt: {
+          type: 'time',
+          sql: 'created_at'
+        },
+        name: {
+          type: 'string',
+          sql: 'name'
+        }
+      }
+    })
+
+    cube(\`Deals\`, {
```

Comment:
This cube is unused by the tests, please remove.

Reply:
Removed in 06b539bf13bffe597f855018d8d80fe383812df3